pkg/controller/status/status.go

```diff
@@ -301,75 +301,67 @@ func (m *statusManager) syncState() bool {
 	defer m.lock.Unlock()
 	progressing := []string{}
 	failing := []string{}
-	numDaemonSets := len(m.daemonsets)
```

Comment:
Some of this isn't really necessary, the num* fields weren't really used and the if len(.... was unnecessary and the cron job loop didn't have it, so I made it consistent.

I noticed this when I went to add the cron job count to the if statement at the bottom